test.sh

```diff
@@ -49,3 +49,13 @@ if [ ! -z ${files} ]; then
 	echo "Boilerplate missing in: ${files}."
 	exit 1
 fi
+
+# Check that cobra docs are up to date
+# This is done by generating new docs and then seeing if they are different than the committed docs
+go run gen_help_text.go
+if [[ $(git diff) ]]; then
```

Comment:
Just thought of this, but could you store this result in a variable? That way you can echo it out in the error message as well.

```shell
files=$(git diff)
if [[ $files ]]; then
  echo "Help text is out of date... $files"
```